frontend/src/views/Settings.vue

```diff
@@ -0,0 +1,141 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+<template>
+  <v-container fluid class="px-6">
+    <v-row>
+      <v-col cols="12" md="6">
+        <v-card class="mt-4">
+          <v-toolbar flat dense class="toolbar-background toolbar-title--text">
+            <v-toolbar-title>Global</v-toolbar-title>
+          </v-toolbar>
+          <v-list>
+            <v-list-item>
+              <v-list-item-content>
+                <v-list-item-title class="text--secondary">Color Scheme</v-list-item-title>
+                <v-btn-toggle v-model="colorSchemeIndex" mandatory dense @click.native.stop class="pt-1">
+                  <v-tooltip top>
+                    <template v-slot:activator="{ on }">
+                      <v-btn small v-on="on">
+                        <v-icon small color="primary">mdi-white-balance-sunny</v-icon>
+                      </v-btn>
+                    </template>
+                    <span>Light Mode</span>
+                  </v-tooltip>
+                  <v-tooltip top>
+                    <template v-slot:activator="{ on }">
+                      <v-btn small v-on="on">
+                        <v-icon small color="primary">mdi-weather-night</v-icon>
+                      </v-btn>
+                    </template>
+                    <span>Dark Mode</span>
+                  </v-tooltip>
+                  <v-tooltip top>
+                    <template v-slot:activator="{ on }">
+                      <v-btn small v-on="on">
+                        <v-icon small color="primary">mdi-brightness-auto</v-icon>
+                      </v-btn>
+                    </template>
+                    <span>Automatically choose theme based on your system settings</span>
+                  </v-tooltip>
+                </v-btn-toggle>
+              </v-list-item-content>
+            </v-list-item>
+            <v-list-item>
+              <v-list-item-content>
+                <v-list-item-title class="text--secondary">Log Level</v-list-item-title>
+                <v-btn-toggle v-model="logLevelIndex" dense class="pt-1">
+                  <v-btn v-for="{ value, text, icon, color } in logLevels"
+                    :key="value"
+                    small
+                    class="text-lowercase text-caption"
+                  >
+                    <v-icon small class="mr-1" :color="color">{{icon}}</v-icon>
+                    {{ text }}
+                  </v-btn>
+                </v-btn-toggle>
+              </v-list-item-content>
+            </v-list-item>
+            <v-list-item>
+              <v-list-item-content>
+                <v-switch
+                  v-model="developerMode"
+                  label="Enable developer mode"
+                ></v-switch>
+              </v-list-item-content>
+            </v-list-item>
```

Comment:
- remove switch
- always show settings button on Avatar menu
- move gardenctl settings from MyAccount to settings
- adapt router link  accordingly https://github.com/gardener/dashboard/blob/0c406ff9d318682c5897b70fd069df32af79575c/frontend/src/components/ShootDetails/GardenctlCommands.vue#L52
- OR: drop gardenctl settings altogether

Reply:
I have moved it to the Settings page